src/app/api/agent/chat/route.ts

```
@@ -0,0 +1,382 @@
+  if (intent.entities.twitterUsernames?.length > 5) {
+    warnings.push('Adding many Twitter sources may increase API usage and costs.');
+  }
+  
+  return warnings;
+}
+
+function getPreviewMessage(intent: any): string {
+  switch (intent.type) {
+    case 'ADD_TWITTER_SOURCE':
+      const usernames = intent.entities.twitterUsernames?.map((u: string) => `@${u}`).join(', ');
+      return `I'll add ${usernames} to your Twitter sources. Please review the changes below.`;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)